aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -292,47 +292,6 @@ void THCTensor_(sigmoid)(THCState* state, THCTensor* self_, THCTensor* src) {
 #endif
 }
 
-void THCTensor_(digamma)(THCState* state, THCTensor* self_, THCTensor* src) {
-  THCAssertSameGPU(THCTensor_(checkGPU)(state, 2, self_, src));
-  if (self_ != src) {
-    THCTensor_(resizeAs)(state, self_, src);
-  }
-  if (!THC_pointwiseApply2<scalar_t, scalar_t>(state, self_, src, TensorDigammaOp<scalar_t, accreal>())) {
```

Comment:
Is it possible to delete `TensorDigammaOp` and `TensorTrigammaOp`?

Reply:
Removed